src/validators/reportValidator.js

```diff
@@ -0,0 +1,17 @@
+export const validateReportBody = (req, res, next) => {
+  const { userId, isAd, detectedAdPhrase, channelId } = req.body || {};
+
+  if (
+    typeof userId !== "number" ||
+    typeof isAd !== "boolean" ||
+    typeof detectedAdPhrase !== "string" ||
+    typeof channelId !== "number"
+  ) {
+    return res.status(400).json({
+      status: 400,
+      error: "올바르지 않은 형식입니다.",
+    });
+  }
```

Comment:
현재 올바르지 않은 형식을 확인하는 조건문이라는건 알겠는데 조건문의 코드가 길어서 이렇게 상수로 분리하는건 어떤가요?

```suggestion
  const isInvalidFormat =
    typeof userId !== 'number' ||
    typeof isAd !== 'boolean' ||
    typeof detectedAdPhrase !== 'string' ||
    typeof channelId !== 'number';
  
  if (isInvalidFormat) {
    return res.status(400).json({
      status: 400,
      error: "올바르지 않은 형식입니다.",
    });
  }
```

Reply:
조건문이 한눈에 보기 어려웠는데 말씀해주신 대로 상수로 분리하니까 가독성이 좋아진 것 같습니다! 좋은 리뷰 감사합니다~ 반영했습니다:)